locals.tf

```
@@ -0,0 +1,27 @@
+
+locals {
+  vpc_id               = var.vpc_id
+  build_security_group = var.security_group_id == "" ? 1 : 0
+
+  # Build ssh ingress if we're both making a security group in this module and
+  # have either a key_name or authorized users url set.
+  build_ssh_ingress = var.security_group_id == "" && (var.key_name != "" || var.authorized_users_url != "") ? 1 : 0
+
+  aws_region = data.aws_region.current.name
+  name       = var.name
+
+  # Data used to track along with the built upstream stacks. I'm not convinced
+  # this is really the right way to do this.
+  ci_stack_data = {
+    buildkite_stack_version = "v5.0.0"
+
+    # This selector is derived from the compiled map in the upstream stack's
+    # AWSRegion2AMI map. This is going to have to be updated to include the
+    # OS type when that lands as a release.
+    ami_selector                   = "buildkite-stack-linux-2020-09-08T23-08-45Z*"
```

```suggestion
    ami_selector                   = "buildkite-stack-${var.operating_system}-*"
```

Why pin it to a date? Might as well use the latest.